web3swift/src/Account/EthereumKeyStorage.swift

```diff
@@ -9,28 +9,66 @@
 import Foundation
 
 public protocol EthereumKeyStorageProtocol {
-    func storePrivateKey(key: Data) throws -> Void
-    func loadPrivateKey() throws -> Data
+    func fetchStoredAddresses() throws -> [String]
+    func storePrivateKey(key: Data, with address: String) throws -> Void
```

Comment:
I think it's simpler if user calls `storePrivateKey` without `address` parameter, as it's derived from the key.
Unless the address param is unrelated to the key, like a tag/name (but in this case we should change the APIs)

Reply:
We store private keys already encrypted in files named by address, and for decryption a password is required that EthrereumKeyStorage should no longer know.